### DISCUSSION TITLE
Workaround for custom fonts for VSCode 1.95++ ?

### DISCUSSION THREAD

**Discussion by OCram85** [coder/code-server#7092] [Q&A]
Hey folks,
did anyone found a workaround for adding custom fonts after `workbenchworkbench.web.main.css` was removed from the codebase?
Previously we could simply add css classes there to embed custom fonts: https://github.com/OCram85/arkanum/blob/master/Dockerfile#L53

##### 🔖 Refs 
- https://github.com/microsoft/vscode/issues/234416

**Comment by code-asher** [comment]
Would it be possible to link the `fonts.css` file in the `workbench.html` file instead?  Inject something like `<link rel="stylesheet" href="{{WORKBENCH_WEB_BASE_URL}}/out/vs/code/browser/workbench/fonts.css">` and then copy `font.css` to that directory.

**Reply by OCram85** [reply]
that is good point. I also found this: src/vs/workbench/browser/media/style.css .
maybe it‘s also possible to load the fonts from there

**Reply by OCram85** [reply]
I guess I could fix it by customizing the `workbench.css` directly: https://github.com/OCram85/arkanum/commit/86d637380726a7211f1cfefbf76a6413404fb051

> [!NOTE]
> Unfortunately I couldn't use the files mention from the [vscode repo directly](https://github.com/microsoft/vscode/issues/234416#issuecomment-2503381905) because I wasn't sure if these should exist in the sources or after building any artifacts.
